README.md

```diff
@@ -23,64 +22,16 @@ Rust bindings for the NumPy C-API.
 - [numpy](https://numpy.org/) installed in your Python environments (e.g., via `pip install numpy`)
   - We recommend `numpy >= 1.16.0`, though older versions may work
 
-**Note:**
-Starting from 0.3, rust-numpy migrated from rust-cpython to PyO3.
-If you want to use rust-cpython, use version 0.2.1 from crates.io.
-
-
-## Python 2 support
-Version 0.5.0 is the last version that supports Python 2.
-
-If you want to compile this library with Python 2, please use 0.5.0 from crates.io.
-
-In addition, you have to add a feature flag in `Cargo.toml` like
-``` toml
-[dependencies.numpy]
-version = "0.5.0"
-features = ["python2"]
-```
-
-You can also automatically specify the Python version in `setup.py`,
-using [setuptools-rust](https://github.com/PyO3/setuptools-rust).
-
-
-## Dependency on ndarray
-
-This crate uses types from `ndarray` in its public API. `ndarray` is re-exported
-in the crate root so that you do not need to specify it as a direct dependency.
-
-Furthermore, this crate is compatible with multiple versions of `ndarray` and therefore depends
-on a range of semver-incompatible versions, currently `>= 0.13, < 0.16`. Cargo does not
-automatically choose a single version of `ndarray` by itself if you depend directly or indirectly
-on anything but that exact range. It can therefore be necessary to manually unify these dependencies.
-
-For example, if you specify the following dependencies
-
-```toml
-numpy = "0.15"
-ndarray = "0.13"
-```
-
-this will currently depend on both version `0.13.1` and `0.15.3` of `ndarray` by default
-even though `0.13.1` is within the range `>= 0.13, < 0.16`. To fix this, you can run
-
-```sh
-cargo update ---package ndarray:0.15.3 --precise 0.13.1
-```
-
-to achieve a single dependency on version `0.13.1` of `ndarray`.
-
 ## Example
 
-
 ### Execute a Python program from Rust and get results
```

Comment:
Just noticed that the examples are the other way around to PyO3's readme, which has Python module first, then Rust program second. Do you think we should consider swapping the order here to match?